fairlearn/reductions/_exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -261,6 +261,6 @@ def _pmf_predict(self, X):
 
```

Comment:
do we want to be doing all these style updates in "reductions" right now? we have at least to existing PRs that touch reductions.